projects/uitdatabank/models/event-image-put.json

```diff
@@ -0,0 +1,26 @@
+{
+  "title": "event.image.put",
+  "type": "object",
+  "description": "Describes the request body to update an image at an [event](./event.json).",
+  "examples": [],
+  "properties": {
+    "description": {
```

Comment:
You could use a `$ref` to existing schemas for both these properties I think? 

- https://github.com/cultuurnet/apidocs/blob/main/projects/uitdatabank/models/common-description-localized.json
- https://github.com/cultuurnet/apidocs/blob/main/projects/uitdatabank/models/common-copyrightHolder.json